models/service/hubspot__tickets.sql

```diff
@@ -0,0 +1,230 @@
+{{ config(enabled=var('hubspot_service_enabled', False)) }}
+
+with ticket as (
+
+    select *
+    from {{ var('ticket') }}
+
+), ticket_pipeline as (
+
+    select *
+    from {{ var('ticket_pipeline') }}
+
+), ticket_pipeline_stage as (
+
+    select *
+    from {{ var('ticket_pipeline_stage') }}
+
+{% if var('hubspot_sales_enabled', true) and var('hubspot_deal_enabled', true) and var('hubspot_ticket_deal_enabled', false) %}
+), ticket_deal as (
+
+    select *
+    from {{ var('ticket_deal') }}
+
+), deal as (
+
+    select *
+    from {{ var('deal') }}
+
+), join_deals as (
+    
+    select
+        ticket_deal.ticket_id,
+        ticket_deal.deal_id,
+        deal.deal_name
+        -- would grab `deal.amount` but the currency could be different per deal 
```

Comment:
Let's remove this comment for now. I agree with you though. If we get requests then we can approach this feature.

Reply:
updated